cmtklib/interfaces/eeglab2fif.py

```diff
@@ -0,0 +1,76 @@
+import numpy as  np
+import pandas as pd
+import mne 
+from nipype.interfaces.base import BaseInterface, BaseInterfaceInputSpec, traits, TraitedSpec
+
+class EEGLAB2fifInputSpec(BaseInterfaceInputSpec):
+	"""Input specification for EEGLAB2fif. """
+
+	eeg_ts_file = traits.List(
+		exists=True, desc='eeg * epochs in .set format', mandatory=True)
+	
+	behav_file = traits.List(
+		exists=True, desc='epochs metadata in _behav.txt', mandatory=True)
+	
+	epochs_fif_fname = traits.File(
+		desc='eeg * epochs in .set format', mandatory=True)
+
+	output_query = traits.Dict(
+		desc='BIDSDataGrabber output_query', mandatory=True)
+	
+	derivative_list = traits.List(
+            exists=True, desc='List of derivatives to add to the datagrabber', mandatory=True)
+
+	
+class EEGLAB2fifOutputSpec(TraitedSpec):
+	"""Output specification for EEGLAB2fif."""
+
+	#fif_ts_file = traits.File(
+	#	exists=True, desc='eeg * epochs in .fif format', mandatory=True)
+
+	output_query = traits.Dict(
+        desc='BIDSDataGrabber output_query', mandatory=True)
+
+	derivative_list = traits.List(
+            exists=True, desc='List of derivatives to add to the datagrabber', mandatory=True)
+
+class EEGLAB2fif(BaseInterface):
+	"""Interface for roitimecourses.svd.
+	Inputs
+	------
+	Outputs
+	-------
+	"""
```

Comment:
Doctring could be improve and an example to show how use the interface should be provided